atd-vze/src/Components/GridExportData.js

```diff
@@ -31,9 +31,22 @@ const StyledSaveLink = styled.i`
   }
 `;
 
-const GridExportData = ({ query, columnsToExport, totalRecords }) => {
+const GridExportData = ({
+  query,
+  columnsToExport,
+  totalRecords,
+  roleSpecificColumns,
+  hasSpecificRole,
```

Comment:
definitely open to other suggestions for these prop names `roleSpecificColumns` and `hasSpecificRole`